src/modules/aws_rds/mappers/rds.ts

```diff
@@ -396,6 +396,17 @@ export class RdsMapper extends MapperBase<RDS> {
       for (const e of es) {
         // if instance has an associated cluster we do not delete it
         if (e.dbCluster) {
+          // check if it exists in the db, or has been deleted due to cascade
+          const cluster = await ctx.orm.findOne(DBCluster, {
+            where: { dbClusterIdentifier: e.dbCluster.dbClusterIdentifier },
+          });
```

Comment:
why don't we call the mapper? `this.module.dbCluster.db.read(id, ctx)`

Reply:
i wanted to avoid any possible caching. So mapper could be affected by caching, and entry could be there, even if the entry has been removed from database right? I mostly want to avoid recreation in case that it has been removed due to cascade

Reply:
I see, you could avoid it by deleting the cache first before calling the mapper. This is definitely a change we need to add to the mappers call, the possibility to pass a `force` or `no-cache` flag.